frontend/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "sharespots",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "build-watch": "tsc -w",
```

Comment:
run this command via `npm run build-watch` to have it watch for changes and compile your Js as you are changing it